Mlem/App/Views/Shared/FancyScrollView.swift

```diff
@@ -18,17 +18,21 @@ struct FancyScrollView<Content: View>: View {
     
     @ViewBuilder var content: () -> Content
     @Binding var isAtTop: Bool
+    @Binding var scrollToTopTrigger: Bool
```

Comment:
Could we use the existing `isAtTop` binding for this?

Reply:
I don't think so--I tried an approach where it would scroll to top when `isAtTop` is set to `true`; this worked fine internally to handle the reselect action, but it wouldn't trigger from the binding. It might be possible, but keeping them separate keeps things stable. I've added a TODO comment to flag the idea for further investigation.